apel/__init__.py

```diff
@@ -16,3 +16,11 @@
    @author Konrad Jopek, Will Rogers
 '''
 __version__ = (1, 9, 1)
+
+# Define the database schema version which this version of the code
+# is expecting to use.
+# The two numbers need not necessarily be in sync.
```

Comment:
maybe this comment could be expanded to included what is meant by "in sync", I asusme it's `==`? What happens if they aren't in sync.